pkg/nvidia/Dockerfile

```diff
@@ -0,0 +1,105 @@
+# syntax=docker/dockerfile-upstream:1.5.0-rc2-labs
+
+# Copyright (c) 2024 Zededa, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+FROM lfedge/eve-alpine:82df60e43ab9f8c935584b8c7b4d0a4b0271d608 as build-base
```

Comment:
What about bumping the other packages as well?
```
find ./ -name Dockerfile -exec perl -pnibak -e 's/lfedge\/eve-alpine(\S+)/lfedge\/eve-alpine:82df60e43ab9f8c935584b8c7b4d0a4b0271d608/' "{}" \;
```

Reply:
As stated in the PR's description, it's going to be done once this one is approved. Now that is approved, I will open another PR to bump up all other packages...

Reply:
Thx, fine.